knowledge-base/tooltip-callout-position.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Hide the Tooltip Callout or Change its Position
+description: Learn how to customize the appearance and behavior of the Tooltip small arrow, including adjusting the callout position and hiding it when necessary.
+type: how-to
+page_title: Hide the Tooltip Callout or Change its Position
+slug: tooltip-kb-callout-position
+tags: tooltip, styling, callout
+ticketid: 1628953, 1565205
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>Tooltip for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+This Knowledge Base article answers the following questions:
+
+* How to customize the Tooltip callout ?
+* How to change the positon of the Tooltip callout ?
+* How to modify the position of the small arrow of a Tooltip ?
+* How to hide the Tooltip callout ?
+
+## Solution
+
+1. Set a custom CSS class to the Tooltip through the `Class` parameter. This configuration will allow you to target specific Tooltip instances.
```

Comment:
Present Simple is usually preferable over other tenses unless it modifies the meaning of the sentence. See [here](https://docs.telerik.com/style-guide/basic-rules-and-guidelines#use-simple-tenses).

```suggestion
1. Set a custom CSS class to the Tooltip through the `Class` parameter. This configuration allows you to target specific Tooltip instances.
```